frontstage/routes.md

```diff
@@ -0,0 +1,136 @@
+# ras frontstage routes
+
+This page documents the ras frontstage routes that can be hit.
+
+## Passwords endpoints
+
+To reset a password
+`passwords/reset-password/<token>`
+* POST request to this endpoint allows you to reset your password
+* GET request to this endpoint allows you to get the password reset form
+* `token` is used to be able to access the reset page
+
+`passwords/reset-password/check-email`
+* GET request to this endpoint is used to render a check e-mail template.
+* Don't think this endpoint is used.
+
+`passwords/reset-password/confirmation`
+* GET request to this endpoint renders confirmation page that password's been changed.
```

Comment:
I'd change `password's` to `password has`.  `password's` implies belonging of password, and as a contraction would normally mean `password is` not `password has`.